src/util/execute-lifecycle-script.js

```diff
@@ -366,6 +366,7 @@ export async function execCommand({
         err.EXIT_SIGNAL
           ? reporter.lang('commandFailedWithSignal', err.EXIT_SIGNAL)
           : reporter.lang('commandFailedWithCode', err.EXIT_CODE),
+        String(err.EXIT_CODE),
```

Comment:
A better fix would be to create another `ProcessTermError` with the proper text, and forward the `EXIT_SIGNAL` and `EXIT_CODE` to it. I don't want `code` to be the same as the exit code (`code` is meant to be `EPERM`, `ENOENT`, etc).